components/SpendingsCounter.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+
+export default function SpendingsCounter({spendingInput, categoryName}) {
+  const allSpendingsValues = spendingInput
+    .filter(spendingInput => spendingInput.category === categoryName)
+    .reduce((allSpendingsValues, currentSpending) => {
+      return allSpendingsValues + Number.parseFloat(currentSpending.amount);
+    }, 0);
+
+  const totalSpendingsValue = allSpendingsValues;
```

Comment:
Why you don't use allSpendingsValues directly where you use totalSpendingsValue?

Reply:
Ok, I see now,` totalSpendingsValue` it's superfluous. I'm adjusting it.